packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -195,9 +195,13 @@ class DatePicker extends React.Component {
      */
     onWeekSelect: PropTypes.func,
     /**
-     * A callback function to execute when the mouse cursor leaves the element.
+     * **Deprecated**, A callback function to execute when the mouse cursor leaves the element. Resolves to `onDismiss`.
```

Comment:
I'd add a note that onDismiss will be used before onClickOutside if both are provided.

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1224/commits/3ccf27db5236ebc98c306d903e3ee9329eb169df.